l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol

```diff
@@ -36,7 +36,7 @@ contract MailboxFacet is ZkSyncHyperchainBase, IMailbox {
     string public constant override getName = "MailboxFacet";
 
     /// @dev Era's chainID
-    uint256 public immutable ERA_CHAIN_ID;
+    uint256 internal immutable ERA_CHAIN_ID;
```

Comment:
are you sure? it is not too big of a deal, but it is just easier to double check the correctness of values when they are public

Reply:
I am 99% sure, as it is a diamond facet, should not get a public view method, no?